src/com/rcv/Logger.java

```diff
@@ -1,37 +1,38 @@
-/**
- * Created by Jonathan Moldover and Louis Eisenberg
- * Copyright 2018 Bright Spots
- * Purpose: Wrapper for console and file logging functions
- * All logging and audit output should use these methods to ensure output goes into audit file
- * Version: 1.0
+/*
```

Comment:
Here and elsewhere: the style guide says that each of the internal lines in a multi-line comment should start with an asterisk.

https://google.github.io/styleguide/javaguide.html#s4.8.6-comments

Reply:
Cool, fixing this, but still changing /** to /* because the former indicates a docstring comment and isn't recommended by the style guide for normal block comments.

I also loaded the Google style guide config into IntelliJ from here: https://github.com/google/styleguide#google-style-guides